### ISSUE TITLE
Error in Mac build: Timezone is not one of enum values: null

### ISSUE THREAD

**Issue by rly** [NeurodataWithoutBorders/nwb-guide#842]
### Describe the issue

<img width="978" alt="image" src="https://github.com/NeurodataWithoutBorders/nwb-guide/assets/310197/1da13826-6999-479c-ac57-a92f3242cc62">

I was following the single session tutorial from the beginning. The box was auto-filled. I tried to click Next to proceed and got this error and cannot proceed.

Console:
```
index-178c0fa4.js:30585 Uncaught (in promise) Error: Timezone is not one of enum values: null.
    at JSONSchemaForm.throw (index-178c0fa4.js:30585:11)
    at JSONSchemaForm.validate (index-178c0fa4.js:30638:19)
    at async GuidedPreform.beforeSave (index-178c0fa4.js:158135:5)
    at async GuidedPreform.save (index-178c0fa4.js:132193:7)
    at async GuidedFooter.onNext [as __onNext] (index-178c0fa4.js:158140:7)
```

Earlier in the console, I see:
```
Uncaught (in promise) TypeError: Cannot read properties of null (reading 'split')
    at index-178c0fa4.js:126114:33
    at Array.reduce (<anonymous>)
    at index-178c0fa4.js:126113:51
```

That code points to:
```
timezoneSchema.enumLabels = filteredTimezones.reduce((acc, tz) => {
      const [_2, ...other] = tz.split("/");
      acc[tz] = other.map((part) => header(part)).join(" — ");
      return acc;
    }, {});
```

It seems like the timezones are not being loaded by the time the enumLabels are being set.

If I clear the box, I see an empty dropdown.

On a local build from the main branch (`npm start`), I do not encounter this error.

On a local build from the .app file (after `npm run deploy:mac` and aborting while code-signing), strangely, I also do not encounter this error...

I will keep trying to reproduce this locally.

### Steps to Reproduce

Download latest mac arm64 test 1.0 release and run through tuturial

### Operating System

Mac OS with Mac M1

### GUIDE Version

1.0 test

### Code of Conduct

Yes

### Did you confirm this issue was not already reported?

Yes

**Comment by rly** [comment]
The API endpoint `http://localhost:4242/system/all_timezones` returns `[]` and `http://localhost:4242/system/local_timezone` returns `null`.

We may need to add this to `nwb-guide.spec`:
```
tmp_ret = collect_all('tzdata')
datas += tmp_ret[0]; binaries += tmp_ret[1]; hiddenimports += tmp_ret[2]
```

Will test after #841 is good.

**Comment by CodyCBakerPhD** [comment]
I mean, isn't `zoneinfo` supposed to be a built-in after 3.9?

**Comment by rly** [comment]
It is, but there might be an issue with finding and packaging resource files from that module. https://stackoverflow.com/questions/67711882/exchangelib-and-pyinstaller-zoneinfo-tzdata-utc-issue

It's possible that because the import is within a function, pyinstaller doesn't know to add those files. I'm not sure.

**Comment by rly** [comment]
Fixed by #843
